modules/ROOT/pages/deployment/services/deployment-considerations.adoc

```diff
@@ -0,0 +1,25 @@
+= Deployment Considerations
+:toc: right
+
+:nats-clustering-url: https://docs.nats.io/running-a-nats-service/configuration/clustering
+:nats-helm-url: https://docs.nats.io/running-a-nats-service/nats-kubernetes/helm-chart
+
+:description: When using container orchestration, some considerations should be taken into account when deploying services with respect to scaling.
+
+== Introduction
+
+{description}
+
+== Scaling of Services
+
+Though the majority of services can be scaled by starting them multiple times, this is not always possible for some of them mostly due to technical limitations when targeting High Availability or scaled installations.
+
+* The xref:{s-path}/search.adoc[Search service] cannot be started multiple times. The use in HA or scaled installations is therefore limited. There is, for the time being, no replacement planned.
+
+* The xref:{s-path}/idm.adoc[IDM service] cannot be started multiple times and should not be used in HA / scaled installations, but replaced instead by an LDAP server that supports HA / scaling. Even if no HA / scaling is needed, the IDM service is only supported for a limited amount of users. Consider the limitation up to approximately 200 users.
```

Comment:
```suggestion
* The xref:{s-path}/idm.adoc[IDM service] cannot be started multiple times and should not be used in HA / scaled installations, but replaced instead by an LDAP server that supports HA / scaling. If no LDAP server is available or no HA / scaling is needed, the IDM service can still be used in production environments but does by nature not cover all capabilities a professional LDAP server provides.
```
Relaxing the statement. I guess that a limiting number may never be sufficient but I strongly believe that any company that has more than a couple of users has in its backend an LDAP server. @dragotin what is your POV?

Reply:
sounds good!